AppOne/AppOne/View/HomeVC.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  HomeVC.swift
+//  AppOne
+//
+//  Created by Asapu, Ajaikumar on 19.11.22.
+//
+
+import UIKit
+
+final class HomeVC: UIViewController {
+    private lazy var brandsCollection: UICollectionView = {
+        let collectionV = UICollectionView(
+            frame: view.bounds,
+            collectionViewLayout: UICollectionViewCompositionalLayout.list(using: UICollectionLayoutListConfiguration(appearance: .plain)))
+        collectionV.translatesAutoresizingMaskIntoConstraints = false
+        return collectionV
+    }()
+
+    private lazy var homeVC = HomeViewModel()
+
+    private lazy var dataSource: UICollectionViewDiffableDataSource<SectionTitle, Brand> = {
+        let cellRegistration = UICollectionView.CellRegistration<UICollectionViewListCell, Brand> { cell, index, res  in
+            var contentConfig = cell.defaultContentConfiguration()
+            contentConfig.text = res.name
+            contentConfig.secondaryText = res.headQuarters
+//            contentConfig.image = UIImage(named: "Flower.png")
```

Comment:
why did you comment it here? should work. right?

Reply:
no not working, only system image working but custom image shows too big

Reply:
that can be changed over imageprovider